web-api/src/main/java/com/musinsa/shop/webapi/controller/CategoryApiController.java

```diff
@@ -0,0 +1,49 @@
+package com.musinsa.shop.webapi.controller;
+
+import com.musinsa.shop.domain.category.CategoryDto;
+import com.musinsa.shop.webapi.service.CategoryService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.hateoas.server.mvc.WebMvcLinkBuilder;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+
+import static com.musinsa.shop.domain.support.CategoryValidGroups.Create;
+import static com.musinsa.shop.domain.support.CategoryValidGroups.Update;
+
+@RequiredArgsConstructor
+@RequestMapping("api/v1/mshop")
```

Comment:
여기에 categories 까지 넣으면 좋을듯..!

```suggestion
@RequestMapping("api/v1/mshop/categories")
```

Reply:
url 중복 제거